client/extensions/woocommerce/state/sites/orders/test/reducer.js

```diff
@@ -80,33 +80,39 @@ describe( 'reducer', () => {
 			const action = {
 				type: WOOCOMMERCE_ORDERS_REQUEST,
 				siteId: 123,
-				page: 1,
+				query: {
+					page: 1,
+				},
 			};
 			const newState = isQueryLoading( undefined, action );
-			expect( newState ).to.eql( { '{page:1}': true } );
+			expect( newState ).to.eql( { '{}': true } );
```

Comment:
The `{}`? Means it's the default query, page=1 status=any-- [`getNormalizedOrdersQuery` filters out](https://github.com/Automattic/wp-calypso/blob/update/order-state-use-query/client/extensions/woocommerce/state/sites/orders/utils.js#L20) any variables that are the same as the default. I think this is to avoid having excessively large keys, that we're only storing data by what is different (I grabbed this from the core posts state).